js/script.js

```diff
@@ -6,7 +6,7 @@ var playerOneName = document.querySelector('.player1-name');
 var playerTwoName = document.querySelector('.player2-name');
 var playerThreeName = document.querySelector('.player3-name');
 var startScreen = document.querySelector('.start-screen');
-
+var categoryTitles = document.querySelectorAll('.dan');
```

Comment:
As I mentioned before, I would rename this class to something more appropriate before you forget.